src/core/models.py

```diff
@@ -863,19 +863,25 @@ def has_missing_image_files(self):
                 # attempt to pull a URL from the specified attribute
                 url = os.path.basename(val.get(attribute, None))
 
-                try:
-                    try:
-                        self.images.get(original_filename=url)
-                    except File.MultipleObjectsReturned:
-                        self.images.filter(original_filename=url).first()
-                except File.DoesNotExist:
+                if show_all:
                     missing_elements.append(url)
+                else:
+                    try:
+                        try:
+                            self.images.get(original_filename=url)
+                        except File.MultipleObjectsReturned:
+                            self.images.filter(original_filename=url).first()
+                    except File.DoesNotExist:
+                        missing_elements.append(url)
 
         if not missing_elements:
             return []
         else:
             return missing_elements
 
+    def all_missing_images(self):
+        return self.has_missing_image_files(show_all=True)
```

Comment:
I don't understand this bit, if I'm reading this correctly, it returns all images declared in the XML, not just the missing ones. Is that right? perhaps a comment would help.

Reply:
Its a poorly worded name. It should probably be `all_images` as it returns all rather than thos missing.